breaktheinternet/js/app.js

```diff
@@ -47,31 +47,36 @@ $(function(){
   });
 
   // submit form
-  $('form').submit(function(e){
-    e.preventDefault();
-    
-    var $form = $(this);
-    var params = $form.serialize();
-    
-    $('.form-error').hide();
-    $form.find('input').attr('disabled', 'disabled');
-
-    var $button = $form.find('button')
-    var buttonText = $button.text()
-    $button.text('Saving...');
+  var loc = window.location.href;
     
-    $.post($form.attr('action'), params, function(response) {
-      if (response.data) {
-        showModal();
-      }
-      else {
-        $('.form-error').show();
-      }
-
-      $form.find('input').removeAttr('disabled');
-      $button.text(buttonText);
+    $('form').submit(function(e){
+      
+      if (loc.indexOf('org=dp') === -1) {
+        e.preventDefault();
+        var $form = $(this);
+        var params = $form.serialize();
+        console.log(params)
+        $('.form-error').hide();
+        $form.find('input').attr('disabled', 'disabled');
+        
+        var $button = $form.find('button')
+        var buttonText = $button.text()
+        $button.text('Saving...');
+        
+        $.post($form.attr('action'), params, function(response) {
+          console.log(response)
```

Comment:
remove

Reply:
when I remove the if statement the page no longer redirects, I get CORS issues and the button stays disabled with the text 'Saving'.

Reply:
oh i meant remove the console.log

Reply:
all the console.logs

Reply:
all removed